### PR TITLE
fix doc for yum_repository.py.

### PR DESCRIPTION
**Pull request by rnoma4696**
The description has been replaced with proxy_password and proxy_username.

##### SUMMARY
The descreption has been replaced with proxy_password and proxy_username 

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
ansible/lib/ansible/modules/packaging/os/yum_repository.py

##### ADDITIONAL INFORMATION
none

### CODE REVIEW

**Review by Akasurde — CHANGES_REQUESTED** [review]
@46gorilla Thanks for the contribution. Could you please add a changelog entry so that we can backport this PR ? You can find details about creating a changelog entry [here](https://docs.ansible.com/ansible/latest/community/development_process.html#changelogs-how-to)? Ping me once you are done. Thanks.

**Review by Akasurde — APPROVED** [review]

File: changelogs/fragments/59068-fix doc for yum_repository.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- fix doc for yum_repository.py (https://github.com/ansible/ansible/pull/59068)
```
Comment (Akasurde) [inline changelogs/fragments/59068-fix doc for yum_repository.py]:
```suggestion
- Fix doc for proxy_username and proxy_password in yum_repository.py (https://github.com/ansible/ansible/pull/59068).
```

Change extension of file to `yml` from `py`

**Review by Akasurde — CHANGES_REQUESTED** [review]

**Comment by 46gorilla** [comment]
Please ask a question because I have little experience.
Does PR to fix a document need a changelog?
I have sent PR twice , but I didnt add changelog.
Is a change log "must" or "better" with PR to fix doc?

**Comment by Akasurde** [comment]
@46gorilla Yes, changelog entry allows users to identify what changed in the given release. Since this is a documentation bug and will be backported, changelog entry is required. 

Please rename changelog entry file to `changelogs/fragments/59068-fix_doc_for_yum_repository.yml`, currently it is `changelogs/fragments/59068-fix doc for yum_repository.py`.

**Comment by 46gorilla** [comment]
Thank you for answering.
And I renamed the changelog file.

**Review by Akasurde — APPROVED** [review]

**Pull request merged**